api/all/src/main/java/io/opentelemetry/api/internal/ImmutableSpanContext.java

```diff
@@ -54,6 +54,8 @@ private static AutoValue_ImmutableSpanContext createInternal(
    * @param traceFlags the trace flags of the {@code SpanContext}.
    * @param traceState the trace state for the {@code SpanContext}.
    * @param remote the remote flag for the {@code SpanContext}.
+   * @param skipIdValidation pass true to skip validation of trace ID and span ID as an optimization
+   *     in cases where they are known to have been already validated
```

Comment:
Nit: all param descriptions end with a dot
```suggestion
   *     in cases where they are known to have been already validated.
```